pennylane/_device.py

```diff
@@ -221,15 +222,16 @@ def execute(self, queue, observables, parameters={}):
             self.pre_measure()
 
             for obs in observables:
-                if obs.return_type == "expectation":
+                if obs.return_type == Expectation:
```

Comment:
A minor comment, but it is usually best practice to compare enums by identity, see https://docs.python.org/3/library/enum.html#comparisons
```suggestion
                if obs.return_type is Expectation:
```

Reply:
Thanks for the valid point! I have changed it accordingly.